### PR TITLE
draft for approval

### PR DESCRIPTION
**Pull request by KennyPaul**

### CODE REVIEW

File: meeting-minutes/2024/2024-09-25-TAC.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+* Jason Lee - GSA
+* JP Lomas (Die QRL Stiftung)
+* Nicklas Körtge
+* Basil Hess
+
+## Agenda
+Added project updates
+* Introduction (as needed) (all)
+* OQS
+* PQPC
```
Comment (brian-jarvis-aws) [inline meeting-minutes/2024/2024-09-25-TAC.md]: PQPC --> PQCP

File: meeting-minutes/2024/2024-09-25-TAC.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+* ICMC PQ Day and Happy Hour recap (all)
+* Tooling WG vote results and next steps (all, Nicklas)
+* AWS QKD (Brian)
+* Lifecycle doc pending issues (all)
+* Any New business
+* Action item review
+---
+### OQS update (Basil)
+* liboqs release
+* Implementation of kyber (sp) & mlkem(sp)
```
Comment (brian-jarvis-aws) [inline meeting-minutes/2024/2024-09-25-TAC.md]: Kyber, ML-KEM

File: meeting-minutes/2024/2024-09-25-TAC.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+* Signature keys - disabled by default
+* Many non functional improvemens like moving things to GHI
```
Comment (brian-jarvis-aws) [inline meeting-minutes/2024/2024-09-25-TAC.md]: improvements

File: meeting-minutes/2024/2024-09-25-TAC.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+* OpenSSL provider being updated
+--
+* other project work in progress
+* Waiting for liboqs update for interoperation 
+* Discussion now turning to a V1 release.
+___
+### PQCP update (Nigel)
+* TSC meeting tomorrow
+* libjade(sp) looking to implementation for alignment
```
Comment (brian-jarvis-aws) [inline meeting-minutes/2024/2024-09-25-TAC.md]: Spelling is correct

File: meeting-minutes/2024/2024-09-25-TAC.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+* PR open for krystals(sp) please look at the pr
```
Comment (brian-jarvis-aws) [inline meeting-minutes/2024/2024-09-25-TAC.md]: Assuming this is referring to CRYSTALS. ([ref](https://pq-crystals.org))

File: meeting-minutes/2024/2024-09-25-TAC.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+___
+### Tooling WG and SonarCube CBOM plugin submission (Nicklas)
+* WG creation approved 
+* Nicklas Körtge designated as the interim leader
+* cbomkit project
+* cbomkit-theia project
+* ACTION Kenny work with Nicklas on logistics
+___
+### Brian potential AWS tooling project
+* collegeag QKD quant key dist
```
Comment (brian-jarvis-aws) [inline meeting-minutes/2024/2024-09-25-TAC.md]: Colleague
QKD = Quantum Key Distribution

File: meeting-minutes/2024/2024-09-25-TAC.md (hunk 1 above)
Comment (planetf1) [inline meeting-minutes/2024/2024-09-25-TAC.md]:
```suggestion
* PQCP
```

File: meeting-minutes/2024/2024-09-25-TAC.md (hunk 2 above)
Comment (planetf1) [inline meeting-minutes/2024/2024-09-25-TAC.md]:
```suggestion
* Implementation of Kyber & ML-KEM
```

File: meeting-minutes/2024/2024-09-25-TAC.md (hunk 3 above)
Comment (planetf1) [inline meeting-minutes/2024/2024-09-25-TAC.md]:
```suggestion
* Many non functional improvements like moving things to GHI
```

File: meeting-minutes/2024/2024-09-25-TAC.md (hunk 4 above)
Comment (planetf1) [inline meeting-minutes/2024/2024-09-25-TAC.md]:
```suggestion
* libjade looking to implementation for alignment
```

File: meeting-minutes/2024/2024-09-25-TAC.md (hunk 5 above)
Comment (planetf1) [inline meeting-minutes/2024/2024-09-25-TAC.md]:
```suggestion
* PR open for merging PQCrystals code : please look at the pr in mlkem-c-generic
```

File: meeting-minutes/2024/2024-09-25-TAC.md (hunk 6 above)
Comment (planetf1) [inline meeting-minutes/2024/2024-09-25-TAC.md]:
```suggestion
* Colleague - QKD (Quantum Key Distribution)
```

**Review by planetf1 — CHANGES_REQUESTED** [review]
See proposed changes, mostly based on @brian-jarvis-aws review

**Review by brian-jarvis-aws — APPROVED** [review]

**Comment by SWilson4** [comment]
@KennyPaul just checking—did you mean to close this PR or merge it?

**Pull request closed without merging**